core/io/file_access_network.cpp

```diff
@@ -500,8 +500,11 @@ uint64_t FileAccessNetwork::_get_modified_time(const String &p_file) {
 void FileAccessNetwork::configure() {
 
 	GLOBAL_DEF("network/remote_fs/page_size", 65536);
+	ProjectSettings::get_singleton()->set_custom_property_info("network/remote_fs/page_size", PropertyInfo(Variant::INT, "network/remote_fs/page_size", PROPERTY_HINT_RANGE, "1, 65536, 1, or_greater")); //is used as denominator and can't be zero
```

Comment:
The max doesn't give much room for increasing the page size, given that the default value is already 65536. Maybe it should be a bit more?